featurebyte/query_graph/sql/aggregator/window.py

```diff
@@ -529,8 +542,7 @@ def merge_tiles_order_independent(
         self,
         req_joined_with_tiles: Select,
         inner_group_by_keys: list[Expression],
-        merge_exprs: list[str],
```

Comment:
these fields aren't needed anymore as they are encapsulated by the `GroupbyColumn` struct